gatsby/src/components/SiteHeader/SiteHeader.test.tsx

```diff
@@ -95,77 +100,86 @@ describe("Header", () => {
 		);
 	});
 
-	it("should append a label for keyword, topic and scenario suggestions", async () => {
+	it("should use gatsby navigate with encoded query when submitting search form", async () => {
 		renderWithRouter(<SiteHeader />);
 
-		userEvent.type(await waitFor(() => screen.findByRole("combobox")), "dia");
+		const searchBox = (await screen.findByRole("combobox")) as HTMLInputElement;
+		searchBox.value = "diabetes 20%";
 
-		await waitFor(() => {
-			expect(
-				screen.queryAllByRole("option").map((n) => n.textContent)
-			).toStrictEqual([
-				"Diabetes - type 1 (CKS topic)",
-				"Diazepam (CKS search)",
-				"How should I manage cardiovascular risk in an adult with type 2 diabetes? (CKS topic - scenario)",
-			]);
-		});
-	});
+		fireEvent.submit(await screen.findByRole("search"));
 
-	it("should use global autocomplete suggestion template when provided instead of default", async () => {
-		const autocompleteSuggestionTemplate = jest.fn(
-			(suggestion) => `<p>${suggestion.Title}</p>`
-		);
-		window.autocompleteSuggestionTemplate = autocompleteSuggestionTemplate;
+		expect(navigate).toHaveBeenCalledWith("/search/?q=diabetes%2020%25");
+	});
 
-		renderWithRouter(<SiteHeader />);
+	describe("Search box", () => {
+		it("should append a label for keyword, topic and scenario suggestions", async () => {
+			renderWithRouter(<SiteHeader />);
+
+			userEvent.type(await waitFor(() => screen.findByRole("combobox")), "dia");
+
+			await waitFor(() => {
+				expect(
+					screen.queryAllByRole("option").map((n) => n.textContent)
+				).toStrictEqual([
+					"Search for dia",
+					"Diabetes - type 1 (CKS topic)",
+					"Diazepam (CKS search)",
+					"How should I manage cardiovascular risk in an adult with type 2 diabetes? (CKS topic - scenario)",
+				]);
+			});
+		});
 
-		userEvent.type(await waitFor(() => screen.findByRole("combobox")), "dia");
+		it("should use global autocomplete suggestion template when provided instead of default", async () => {
+			const user = userEvent.setup();
+			const autocompleteSuggestionTemplate = jest.fn(
+				(suggestion) => `<p>${suggestion.Title}</p>`
+			);
+			window.autocompleteSuggestionTemplate = autocompleteSuggestionTemplate;
+
+			renderWithRouter(<SiteHeader />);
+
+			user.type(await waitFor(() => screen.findByRole("combobox")), "dia");
+
+			//TODO need to work out why there are 6 calls being made for the suggestion template versus 3 before GN upgrade
```

Comment:
Have we made a ticket for this?

Reply:
yes

Reply:
https://nicedigital.atlassian.net/browse/CKS-701